content/reference/current/plugins/github_actions/includes/use-case-binary-repo.gradle.md

```diff
@@ -0,0 +1,55 @@
+```yaml
+name: Run Gatling Enterprise Simulation
+
+# Execute the workflow on each push to the main branch
+on:
+  push:
+    branches:
+      - main
+
+# Here we use concurrency to cancel previous executions if they are still
+# ongoing. Useful to avoid running the same simulation several times
+# simultaneously if we push several code changes in a short time.
+# See https://docs.github.com/actions/using-jobs/using-concurrency.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
Did you try it for a long run?
I mean, does it correctly stop the older and start the newer?

Reply:
I've done a couple tests with a self-hosted instance and with a `workflow_dispatch` workflow, launching a new one does cancel the previous one, which stops the ongoing simulation, before the new workflow starts executing.